.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Please remove the Visual Studio configuration. If you need it in the directory, you can add it to `.gitignore`.